dyc/main.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, config, details=None, placeholders=False):
         self.config = config
         self.placeholders = placeholders
 
-    def process_methods(self, diff_only=False, changes=[]):
+    def process_methods(self, project_directory="", diff_only=False, changes=[]):
```

Comment:
Added a field for project_directory and passed it to the method builder from which I determine the correct path to print.